sbt/src/sbt-test/source-dependencies/macro-annotation/project/Build.scala

```diff
@@ -0,0 +1,45 @@
+import sbt._
+import Keys._
+
+object BuildSettings {
+  val paradiseVersion = "2.0.1"
+  val buildSettings = Defaults.defaultSettings ++ Seq(
+    version := "1.0.0",
+    scalacOptions ++= Seq(""),
+    scalaVersion := "2.11.4",
+    resolvers += Resolver.sonatypeRepo("snapshots"),
+    resolvers += Resolver.sonatypeRepo("releases"),
+    addCompilerPlugin("org.scalamacros" % "paradise" % paradiseVersion cross CrossVersion.full),
+    incOptions := incOptions.value.withNameHashing(true)
+  )
+}
+
+object MyBuild extends Build {
+  import BuildSettings._
+
+  lazy val root: Project = Project(
+    "root",
+    file("."),
+    settings = buildSettings ++ Seq(
+      run <<= run in Compile in core
+    )
+  ) aggregate(macros, core)
+
+  lazy val macros: Project = Project(
+    "macros",
+    file("macros"),
+    settings = buildSettings ++ Seq(
+      libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-reflect" % _),
```

Comment:
I thought that `<+=` got deprecated (or at least new code should use it with care) since `scalaVersion.value` works and looks so much "better".